docs/development/How to build AL distribution.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'AL Building'
+---
+
+# How to build AlmaLinux OS distribution on one's own
+
+## Hardware and software requirements to a final machine 
+
+* AlmaLinux 9
+* 6+ CPU cores
+* 8+ GB RAM
+* 300+ GB free storage space
+
+## Deploying a Pungi node
```

Comment:
I'd probably merge the two sections "Deploying a Pungi node" and "Do the following steps to prepare your machine" into a single one, "Setting up your machine" or something like that. Then, explain why we need a pungi node, and finally, put the step by step process as you already have.